### ISSUE TITLE
Does not work with the German version of the game

### ISSUE THREAD

**Issue by TheMechanist902** [JeffRuLz/LEGO-Island-2-Patcher#1]
Since the executables differ it does not work for the German version

**Comment by JeffRuLz** [comment]
Right, this was based on the English releases. I did not create the patches used in this program and I do not know how they were created. 

I recommend you ask on the Rock Raiders United forum for help. 
http://www.rockraidersunited.com/forum/390-modding/

Ask for the German releases' version of these patches:
http://www.rockraidersunited.com/topic/7206-lego-island-2-patches-using-a-hex-editor/

And if you do get the patches, use the new LI2 Mod Manager program instead of this old one: http://www.rockraidersunited.com/topic/7381-li2-mod-manager/
I can help you setup the mod files when the patches are found.

But for now, all I can do is add a message on the front page saying that this is only for the English version of the game.

**Comment by TheMechanist902** [comment]
Hey thank for your help :) I will see to it.
Maybe you can mention your new tool in the readme of the patcher? 
Btw I found it via http://pcgamingwiki.com/wiki/Lego_Island_2:_The_Brickster%27s_Revenge#LEGO_Island_2_Patcher
Thanks for your work anyway. It's nice to see some people are still
attached to such a game :)
Have a nice day

EDIT: OK you already mentioned it. Thanks.
2016-12-22 3:41 GMT+01:00 JeffRuLz <notifications@github.com>:
